custom-vu/src/main/kotlin/jces1209/vu/page/project/ProjectIssueNavigatorPage.kt

```diff
@@ -0,0 +1,9 @@
+package jces1209.vu.page.project
+
+import org.openqa.selenium.WebDriver
+
+interface ProjectIssueNavigatorPage {
+
+    fun openProjectByIndex(index: Int)
```

Comment:
Project is being loaded from project key from memory
Let's make it so
```suggestion
    fun openProject(projectKey: String)
```